dcicutils/s3_utils.py

```diff
@@ -154,6 +161,9 @@ def __init__(self, outfile_bucket=None, sys_bucket=None, raw_file_bucket=None,
                     tibanna_output_bucket = tibanna_output_bucket_from_health_page
                 logger.warning('Buckets resolved successfully.')
             else:
+                # TODO: We require global env bucket to be set everywhere now, so can we throw an error here?
+                #       -kmp 1-Mar-2023
```

Comment:
I think throwing an error is fine at this point.

Reply:
OK, I'm preparing a layered PR that will include error-checking of the form you're approving here. This has been added.